src/ui/EditorNS/editor.cpp

```diff
@@ -231,7 +232,8 @@ namespace EditorNS
     void Editor::setLanguage(const QString &language)
     {
         sendMessage("C_CMD_SET_LANGUAGE", language);
-        if (!m_customIndentationMode)
+        sendMessage("C_CMD_REFRESH");
+	if (!m_customIndentationMode)
```

Comment:
fixKeyboardShortcuts is redundant due to the new custom keyboard shortcuts.
I'll see about fixing indentations after I get off work later. If there's
anything else you can think of let me know.

On Aug 11, 2016 6:25 AM, "Daniel Di Sarli" <notifications@github.com> wrote:

> In src/ui/EditorNS/editor.cpp
> <https://github.com/notepadqq/notepadqq/pull/230#discussion_r74398999>:
>
> > @@ -231,7 +232,8 @@ namespace EditorNS
> >      void Editor::setLanguage(const QString &language)
> >      {
> >          sendMessage("C_CMD_SET_LANGUAGE", language);
> > -        if (!m_customIndentationMode)
> > +        sendMessage("C_CMD_REFRESH");
> > +	if (!m_customIndentationMode)
>
> Can you fix indentation?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/notepadqq/notepadqq/pull/230/files/62e470aac7ccc4023ba1db6ee3da401ed3b8b52d#r74398999>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACvJZS3YQ-HBHyDVfPuPEUKdG7T0gaqAks5qeviNgaJpZM4JdYv0>
> .
>